contrib/filter-repo-demos/git_lint_history.py

```diff
@@ -97,9 +98,6 @@ def is_relevant(filename):
               "the file's extension), then pass this argument"))
 parser.add_argument('command', nargs=argparse.REMAINDER,
         help=("Lint command to run, other than the filename at the end"))
-lint_args = parser.parse_args()
-if not lint_args.command:
-  raise SystemExit("Error: Need to specify a lint command")
```

Comment:
This doesn't match the commit message as far as I can tell; you're not only making a main() but adding other changes as well.  What's the purpose of those changes?  Can they be split into another commit, so they can be independently motivated in their own commit message?